test/cppsuite/test_harness/thread_context.h

```diff
@@ -105,20 +122,19 @@ class thread_context {
     {
         if (!_in_txn && _timestamp_manager->is_enabled()) {
             testutil_check(
-              session->begin_transaction(session, config.empty() ? NULL : config.c_str()));
+              session->begin_transaction(session, config.empty() ? nullptr : config.c_str()));
             /* This randomizes the number of operations to be executed in one transaction. */
             _max_op_count = random_generator::instance().generate_integer(_min_op, _max_op);
             _current_op_count = 0;
             _in_txn = true;
         }
     }
 
-    /* Returns true if the current transaction has been committed. */
-    bool
+    void
     commit_transaction(WT_SESSION *session, const std::string &config)
```

Comment:
I thought we were going to remove the op count stuff and make these functions explicitly begin or commit a transaction?

Reply:
That's correct, this section will be reworked.